public/app/features/trails/otel/api.ts

```diff
@@ -249,7 +249,15 @@ export async function getFilteredResourceAttributes(
   const url = `/api/datasources/uid/${datasourceUid}/resources/api/v1/labels`;
 
   // The match param for the metric to get all possible labels for this metric
-  const metricMatchParam = `${metric}{job=~"${metricResources.jobs.join('|')}",instance=~"${metricResources.instances.join('|')}"}`;
+  const metricMatchTerms = limitOtelMatchTerms([], metricResources.jobs, metricResources.instances);
+
+  let metricMatchParam = `${metric}{${metricMatchTerms.jobsRegex},${metricMatchTerms.instancesRegex}}`;
+
+  if (metricMatchTerms.missingOtelTargets) {
+    console.warn(
+      'Truncating job and instance values for label values GET request. Please consider filtering by more OTel resource attributes.'
+    );
```

Comment:
Wouldn't it be nicer to show a warning notification? 
Also do you think we should report this? To keep track of how many people face this?

Reply:
1. Showing the warning is a good idea, I can look into surfacing this warning.

2. This is a case we have seen with customers with high cardinality Prometheus and using trace metrics. It is relatively a small number of cases, but having some numbers on this could be helpful

Reply:
Showing the warning now in label breakdown and reporting to rudderstack when we have a metric that has too many job and instance labels for the GET request.